.github/workflows/build-and-test.yml

```diff
@@ -44,21 +41,13 @@ jobs:
         with:
           cuda: ${{ matrix.cuda }}
           method: 'network'
-          sub-packages: '["nvcc"]'
```

Comment:
why no nvcc? don't we need it to compile code with CUDA?

Reply:
When set empty installs everything. Only setting `nvcc` will not install `visual_studio_integration` in Windows. Adding both `nvcc` and `visual_studio_integration` fails in Linux because `visual_studio_integration` is not available for Linux.

Reply:
Ideally we should have ["nvcc"] for Linux and ["nvcc", `"visual_studio_integration"`] in Windows

Reply:
We might want to take a look at it in the future, since installing everything gave us some problems regarding disk space in GitHub Action's workers, maybe add a "TODO" if you can so we don't completely forget it, or either open an issue with it, whatever you prefer

Reply:
Done in https://github.com/gigabit-clowns/xmipp4-compute-cuda/issues/23